lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -144,6 +145,12 @@
                 start: function() {
                     Backbone.history.start();
 
+                    $(document).ajaxError(function (event, xhr) {
+                        if (_.contains([401, 500], xhr.status)) {
+                            TeamUtils.showMessage(gettext("Your request could not be completed. Reload the page and try again."));
```

Comment:
401 error message -- keep what we have in line 150: "Your request could not be completed. Reload the page and try again."
500 error message: "Your request could not be completed due to a server problem. Reload the page and try again. If the issue persists, click the Help tab to report the problem."
@peter-fogg @andy-armstrong @explorerleslie OK?